components/server/src/api/prebuild-service-api.ts

```diff
@@ -173,6 +183,10 @@ export class PrebuildServiceAPI implements ServiceImpl<typeof PrebuildServiceInt
                 offset: paginationToken.offset,
             },
             prebuildsFilter,
+            {
+                ...sorting,
+                order: sorting.order ?? "DESC",
+            },
```

Comment:
```suggestion
            sorting,
```

Reply:
@mustard-mh had to do this because of types 😕. `sorting.order` is not narrowed down properly (although I think it should) and therefore thinks it can be undefined.

Reply:
Maybe something wrong on converter `fromSort`